spec/lib/integrations/slack_spec.rb

```diff
@@ -70,7 +69,7 @@
       end
 
       describe 'run result inclusion in text' do
-        it_should_behave_like "Slack notification with a specific text", "Your Rainforest Run <http://example.com|#123> failed. Time to finish: 25 minutes 3 seconds"
+        it_should_behave_like "Slack notification with a specific text", "Your Rainforest Run <http://example.com | Run #123> failed. Time to finish: 25 minutes 3 seconds"
```

Comment:
There is a duplicated "Run" in the text...this one would give: "Your Rainforest Run Run #123 failed...."